wagtail/admin/templates/wagtailadmin/pages/edit.html

```diff
@@ -50,13 +50,15 @@ <h1 class="icon icon-doc-empty-inverse">
                         <div class="dropdown dropup dropdown-button match-width">
                             {% include "wagtailadmin/pages/_preview_button_on_edit.html" with label=preview_label icon=1 %}
                             <div class="dropdown-toggle icon icon-arrow-up"></div>
-                            <ul role="menu">
-                                {% for mode_name, mode_display_name in preview_modes %}
-                                    <li>
-                                        {% include "wagtailadmin/pages/_preview_button_on_edit.html" with mode=mode_name label=mode_display_name %}
-                                    </li>
-                                {% endfor %}
-                            </ul>
+                            <nav aria-label="actions">
```

Comment:
Same as the previous similar example – this should either have no landmark, or have it on the wrapper ul above.

Reply:
I've moved the `nav` to be around the wrapper `ul` as in the previous example.